buku

```diff
@@ -2247,6 +2249,10 @@ class BukuDb:
                 res = convert_bookmark_set(resultset, 'org')
                 count += res['count']
                 outfp.write(res['data'])
+            elif filepath.endswith('.xbel'):
```

Comment:
Please move all logic for xbel below html in the code. Far too many people use html.

Reply:
Ignore. HTMLs can have multiple extensions.